examples/js/loaders/FBXLoader.js

```diff
@@ -3986,43 +3986,51 @@ THREE.FBXLoader = ( function () {
 
 		// Global Shear*Scaling
 		var lParentTM = new THREE.Matrix4();
-		var lLSM;
-		var lParentGSM;
-		var lParentGRSM;
+		var lLSM = new Matrix4();
+		var lParentGSM = new Matrix4();
+		var lParentGRSM = new Matrix4();
+		
+		var lParentTM_inv = new Matrix4().getInverse( lParentTM );
 
 		lParentTM.copyPosition( lParentGX );
-		lParentGRSM = lParentTM.getInverse( lParentTM ).multiply( lParentGX );
-		lParentGSM = lParentGRM.getInverse( lParentGRM ).multiply( lParentGRSM );
+		lParentGRSM.multiply( lParentTM_inv ).multiply( lParentGX );
+		
+		var lParentGRM_inv = new Matrix4().getInverse( lParentGRM );
+		lParentGSM.multiply( lParentGRM_inv ).multiply( lParentGRSM );
 		lLSM = lScalingM;
 
-		var lGlobalRS;
+		var lGlobalRS = new Matrix4();
 		if ( inheritType === 0 ) {
 
-			lGlobalRS = lParentGRM.multiply( lLRM ).multiply( lParentGSM ).multiply( lLSM );
+			lGlobalRS.multiply( lParentGRM ).multiply( lLRM ).multiply( lParentGSM ).multiply( lLSM );
```

Comment:
```js
lGlobalRS.copy( lParentGRM ).multiply( lLRM ).multiply( lParentGSM ).multiply( lLSM );
```
multiple places...

Reply:
> lGlobalRS.copy( lParentGRM )

> lGlobalRS.multiply( lParentGRM )

These two statements are equivalent for `lGlobalRS = I`, so what would be the benefit of using `.copy` over `.multiply`?

Reply:
For the same reason we would use `.copy()` instead of
```js
matrix1.identity().multiply( matrix2 );
```
It is clearer.

Reply:
https://github.com/mrdoob/three.js/commit/809fa4f7f9925f4421acb4eb200250f781c3b0dd